bubble_sort/Python/Bubble_sort.py

```diff
@@ -0,0 +1,9 @@
+def bubble_sort(mas):
+    for i in range(len(mas)):
+        for j in range(len(mas) - 1, i, -1):
+            if mas[j] < mas[j-1]:
+                mas[j], mas[j-1] = mas[j-1], mas[j]
+
+    return mas
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmp_fzqzmf7/bubble_sort/Python/Bubble_sort.py
+++ b/tmp/tmp_fzqzmf7/bubble_sort/Python/Bubble_sort.py
@@ -6,4 +6,5 @@
 
     return mas
 
+
 print(bubble_sort([4, 3, 2, 1]))
```